utils/potential_utf/Cargo.toml

```diff
@@ -5,7 +5,7 @@
 [package]
 name = "potential_utf"
 description = "Unvalidated string and character types"
-version = "0.1.1-dev"
+version = "0.1.2"
```

Comment:
are you sure this is not semver breaking? why was it `-dev`

Reply:
I'll have another look. I don't think it is

Reply:
```
$ cargo semver-checks 
    Building potential_utf v0.1.2 (current)
       Built [   5.137s] (current)
     Parsing potential_utf v0.1.2 (current)
      Parsed [   0.007s] (current)
     Parsing potential_utf v0.1.0 (baseline, cached)
      Parsed [   0.009s] (baseline)
    Checking potential_utf v0.1.0 -> v0.1.2 (minor change)
     Checked [   0.022s] 120 checks: 120 pass, 7 skip
     Summary no semver update required
    Finished [   5.884s] potential_utf
```

I never fully understood our process of using `-dev`, we should probably document that better. I didn't find the `-dev` tags super helpful this time around.

Also I think we've been doing `oldversion-dev` when we should be doing `newversion-dev`

Reply:
Turns out there was a _non_ breaking change to potential_utf hidden in a different commit. Added to changelog.